Regressions/scripts/Vehicles/Airfoils/Clark_y/Clark_y.txt

```diff
@@ -0,0 +1,126 @@
+CLARK Y AIRFOIL
```

Comment:
Aren't these files also in the modules folder?

Reply:
Yes they are. Updated